src/gist.c

```diff
@@ -681,6 +682,10 @@ g_spoint3_consistent(PG_FUNCTION_ARGS)
 	PG_RETURN_BOOL(false);
 }
 
+static double distance_vector_point_3d (Vector3D* v, double x, double y, double z) { 
```

Comment:
Change `Vector3D* v` to `Vector3D *v` in order to match the coding style of pgsphere.